### PR TITLE
Fix repeated options in Property and Profile select.

### PR DESCRIPTION
**Pull request by jimmymadon**
## Summary

Addresses issue:

- #6740 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

**Review by nfmohit — APPROVED** [review]
LGTM, thanks @jimmymadon! 👍

File: assets/js/modules/analytics/components/common/ProfileSelect.js [hunk 1]
```diff
@@ -117,7 +117,10 @@ export default function ProfileSelect( { hasModuleAccess } ) {
 		);
 	}
 
-	if ( ! ga4ReportingEnabled ) {
+	if (
+		! ga4ReportingEnabled &&
+		! profiles.find( ( profile ) => profile.id === PROFILE_CREATE )
```
Comment (techanvil) [inline assets/js/modules/analytics/components/common/ProfileSelect.js:122]: Hey @jimmymadon - I feel a bit pedantic for this, but we do tend to use `some()` in the codebase which returns a boolean, when we don't actually need the array element result of `find()`. So, I'd suggest updating as follows:

```suggestion
		! profiles.some( ( profile ) => profile.id === PROFILE_CREATE )
```

File: assets/js/modules/analytics/components/common/PropertySelect.js [hunk 2]
```diff
@@ -111,7 +111,10 @@ export default function PropertySelect( { hasModuleAccess } ) {
 		);
 	}
 
-	if ( ! ga4ReportingEnabled ) {
+	if (
+		! ga4ReportingEnabled &&
+		! properties.find( ( property ) => property.id === PROPERTY_CREATE )
```
Comment (techanvil) [inline assets/js/modules/analytics/components/common/PropertySelect.js:116]: As above:

```suggestion
		! properties.some( ( property ) => property.id === PROPERTY_CREATE )
```

**Review by techanvil — CHANGES_REQUESTED** [review]
Looks good, @jimmymadon - just a couple of very trivial comments from me. Also, please note I've updated the target branch to `main` as the previous PR for this issue got into the 1.98.0 release.

**Review by techanvil — APPROVED** [review]
LGTM, nice one @jimmymadon!

**Pull request merged**